docs/CLI-Reference.md

```diff
@@ -253,7 +253,8 @@ This file is auto-generated from the output of `cl help -v` and provides the lis
           -v, --verbose                Display verbose output.
 
       kill:
-        Instruct the appropriate worker to terminate the running bundle(s).
+        Instruct the appropriate worker to terminate the bundle(s) in the following states:
+         created, uploading, staged, making, starting, running, preparing, or finalizing.
```

Comment:
Since we can kill bundle before they're even assigned a worker now, let's change the language here to reflect that and maybe something like "terminate the bundle execution if it's being executed or remove it from the execution queue" (but maybe with better wording)